### DISCUSSION TITLE
No more time displayed

### DISCUSSION THREAD

**Discussion by QuintaEssentia** [aclap-dev/video-downloadhelper#558] [Help and Q&A]
### Your question

Hi!
In the new version I find it almost impossible to identify a video from others on the page, because the time of the video is not displayed anymore. I looked through the settings and did not find this option. Is it possible to go back to the previous interface? This one is highly inconvenient. 

### Addon & CoApp version and details

version: 9.0.1.13
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"3d68e65e66f6489f"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\shell\\dwhelper",
    "view_options": {
        "all_tabs": true,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 11
}

**Comment by paulrouget** [comment]
>  Is it possible to go back to the previous interface? 

Yes, in the settings, at the bottom, "use legacy UI".

**Comment by paulrouget** [comment]
> the time of the video is not displayed anymore

Actually, it is displayed in the thumbail:

<img width="336" alt="Screenshot 2024-05-23 at 17 24 15" src="https://github.com/aclap-dev/video-downloadhelper/assets/373579/07e71988-aec7-4d3c-97ac-6702c3a627ef">

**Reply by QuintaEssentia** [reply]
I don't have it:
![image](https://github.com/aclap-dev/video-downloadhelper/assets/34210564/7340de6f-8baf-4ab3-bed4-c58db79b2f71)

**Reply by paulrouget** [reply]
Oh I seee, that's for HTTPMedia videos. I'll look into it, thank you!
